### PR TITLE
Allow to customize Ironic extra parameter for serial number

### PR DESCRIPTION
**Pull request by romcheg**

### CODE REVIEW

File: src/ralph/virtual/management/commands/openstack_sync.py [hunk 1]
```diff
@@ -404,16 +410,18 @@ def _match_nodes_to_hosts(self, nodes):
         )
 
         for node in nodes:
+            node_sn = node.extra[self.ironic_serial_number_param]
+
             try:
                 host = CloudHost.objects.get(host_id=node.instance_uuid)
                 asset = DataCenterAsset.objects.get(
-                    sn=node.extra.get('serial_number')
+                    sn=node_sn
```
Comment (mkurek) [inline src/ralph/virtual/management/commands/openstack_sync.py]: it'd be useful to have an option to customize `DataCenterAsset` field (`sn` right now) as well

Comment (romcheg) [reply src/ralph/virtual/management/commands/openstack_sync.py]: Done

**Review by mkurek — APPROVED** [review]

File: src/ralph/virtual/management/commands/openstack_sync.py [hunk 2]
```diff
@@ -426,11 +441,20 @@ def _match_nodes_to_hosts(self, nodes):
...
-                        node.extra.get('serial_number'),
+                    'Please match Cloud host {} manually.'.format(
+                        node_sn,
                         host.id
                     )
                 )
+            except KeyError:
+                logger.warning(
+                    'Could not get serial number of the Ironic node {} using '
+                    'using {} extra parameter. Please check the configuration '
```
Comment (ghost) [inline src/ralph/virtual/management/commands/openstack_sync.py]: double `using`?

Comment (romcheg) [reply src/ralph/virtual/management/commands/openstack_sync.py]: Good catch!

**Review by ghost — APPROVED** [review]

**Pull request merged**